doc/installationguide.md

```diff
@@ -16,7 +16,7 @@ These are the parameters that can be configured in the global section:
         }
 ```
 
- * If you want to use NGSIv2 (only sending updates for active attributes):
+ * If you want to use NGSIv2 (sending updates for active attributes and NGISv2-based registration):
```

Comment:
Maybe the "(...)" should be removed at all. I think we are near to finalize the last part of NGSIv2 (support in forwarded queries) and if we leave the "(...)" note it could become obsolete soon and we may forget removing it.

Reply:
Fixed with ae87e84e298bbf60882f425713c41f1795209bb4